docroot/modules/humsci/hs_paragraph_types/modules/hs_hero_image/config/install/responsive_image.styles.hero_image.yml

```diff
@@ -15,30 +15,15 @@ image_style_mappings:
     multiplier: 1x
     image_mapping_type: image_style
     image_mapping: hero_large
-  -
-    breakpoint_id: stanford_media.lg
```

Comment:
Don't need the high-density styles?

Reply:
we had some issues with the 2x pixel density due to the size of image they provided

Reply:
Not enough pixels? 

Another approach would be to require pixel dimensions on the fields that use this hero style. Seems like you are sacrificing your end-users with Retina displays over the content author's ability to find hi-res images.

Reply:
Probably want to clean up the derived images and any saved config that used these if you choose to go this way.

Reply:
we'd to to implement some way to restrict what images are allowed to be uploaded/selected in the media library to do that. might be a difficult task but something we could do in another project.

This particular file is a copy from the `config/default` file. so existing sites dont need any cleanup or changes.